CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+Contributing
+============
+
+It's good to hear that you want to contribute to Corral!
+
+There are a number of ways to contribute to Corral. As this document is a little long, feel free to jump to the section that applies to you currently:
+
+* [Feature request](#feature-request)
+* [Bug report](#bug-report)
+* [How to contribute](#how-to-contribute)
+* [Pull request](#pull-request)
+
+Additional notes regarding formatting:
+
+* [Documentation formatting](#documentation-formatting)
+* [Code formatting](#code-formatting)
+
+Feature request
+---------------
+For any feature requests or enhancements to Corral, it is quite likely that you have to go through our [RFC process](https://github.com/ponylang/rfcs). Before opening or submitting any feature requests, please make sure you are familiar with the RFC process and follow the process as required.
```

Comment:
Do we want to require all Corral features + enhancements to go through the Pony RFC process? How about just larger changes, where the threshold can be discussed on Zulip.

Reply:
That is what the language generally says, it doesn't say required. Say "likely". This is the standard boilerplate for all the Pony stuff.